drivers/hmis/spec/system/hmis/form_builder_spec.rb

```diff
@@ -0,0 +1,285 @@
+        fill_in 'Display Text', with: 'This is a <b>displayable item</b>.'
+        find("button[type='submit']").trigger('click')
+        assert_text 'Display: This is a displayable item.'
+
+        find("button[aria-label='Add Text item']").click
+        assert_text 'ADD NEW FORM ITEM Text Item'
+        find("input[name='text']").fill_in(with: 'What is the answer to this question?')
+        find("button[type='submit']").trigger('click')
+        assert_text 'Text: What is the answer to this question?'
+
+        find("button[aria-label='Add Paragraph item']").click
+        assert_text 'ADD NEW FORM ITEM Paragraph Item'
+        find("input[name='text']").fill_in(with: 'What about this long paragraph?')
+        find("button[type='submit']").trigger('click')
+        assert_text 'Paragraph: What about this long paragraph?'
+
+        find("button[aria-label='Add Choice item']").click
+        assert_text 'ADD NEW FORM ITEM Choice Item'
+        find("input[name='text']").fill_in(with: 'Which option?')
+        click_button 'Add Choice'
+        fill_in 'Choice 1', with: 'One'
+        click_button 'Add Choice'
+        fill_in 'Choice 2', with: 'Two'
+        find("button[type='submit']").trigger('click')
+        assert_text 'Choice: Which option?'
+
+        find("button[aria-label='Add Date item']").click
+        assert_text 'ADD NEW FORM ITEM Date Item'
+        find("input[name='text']").fill_in(with: 'What day?')
+        find("button[type='submit']").trigger('click')
+        assert_text 'Date: What day?'
+
+        find("button[aria-label='Add Number item']").click
+        assert_text 'ADD NEW FORM ITEM Number Item'
+        find("input[name='text']").fill_in(with: 'How many?')
+        find("button[type='submit']").trigger('click')
+        assert_text 'Number: How many?'
+
+        find("button[aria-label='Add Checkbox item']").click
+        assert_text 'ADD NEW FORM ITEM Checkbox Item'
+        find("input[name='text']").fill_in(with: 'Yes or no?')
+        find("button[type='submit']").trigger('click')
+        assert_text 'Checkbox: Yes or no?'
+
+        draft.reload
+        expect(draft.definition.dig('item', 1, 'text')).to eq('Brand New Group')
+        expect(draft.definition.dig('item', 2, 'text')).to eq('This is a <b>displayable item</b>.')
+        expect(draft.definition.dig('item', 3, 'text')).to eq('What is the answer to this question?')
+        expect(draft.definition.dig('item', 4, 'text')).to eq('What about this long paragraph?')
+        expect(draft.definition.dig('item', 5, 'text')).to eq('Which option?')
+        expect(draft.definition.dig('item', 6, 'text')).to eq('What day?')
+        expect(draft.definition.dig('item', 7, 'text')).to eq('How many?')
+        expect(draft.definition.dig('item', 8, 'text')).to eq('Yes or no?')
+      end
+
+      it 'reorders items' do
+        assert_text 'Group: Test Custom Assessment'
+        find("div[aria-label='item section_1']").click
```

Comment:
I'm not sure, but I think the test failures here are due to the ci running against a frontend based on the `stable` branch instead of `release-130`. Is there anything we should do to update that?

Reply:
Hmm, yeah.
`BRANCH_NAME: "${{ github.head_ref || github.ref_name }}:${{ github.base_ref || 'stable' }}:stable"`
is resolved as
`BRANCH_NAME: martha/6340-formbuilder-sys-test:stable:stable`

Which isn't right. We want `github.base_ref` to resolve `release-130` since the PR is opened against that. The problem is that this github action is trigger on push (NOT pull_request), so [base_ref](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#github-context) isn't working in the way we want.

I can open a ticket, but as a workaround, you can open a branch called `martha/6340-formbuilder-sys-test` on the frontend and push it, and then re-run the tests on CI.

Reply:
https://github.com/open-path/Green-River/issues/6566

Reply:
Ok, makes sense. Thank you for creating the ticket!